assets/src/dashboard/app/views/editorSettings/adManagement/test/adManagement.js

```diff
@@ -17,27 +17,31 @@
  * Internal dependencies
  */
 import { renderWithProviders } from '../../../../../testUtils';
-import AdNetworkSettings, { TEXT } from '../';
 import { AD_NETWORK_TYPE } from '../../../../../constants';
+import AdManagement, { TEXT } from '../index';
 
-describe('Editor Settings: Ad network settings <AdNetworkSettings />', function () {
+describe('Editor Settings: Ad Management group settings <AdManagement />', function () {
   let adNetwork;
   let mockUpdate;
 
   beforeEach(() => {
     adNetwork = AD_NETWORK_TYPE.NONE;
-    mockUpdate = jest.fn((id) => {
-      adNetwork = id;
-    });
+    mockUpdate = jest.fn();
   });
 
   afterEach(() => {
     adNetwork = AD_NETWORK_TYPE.NONE;
   });
 
-  it('should render ad network settings and helper text by default', function () {
-    const { getByText } = renderWithProviders(
-      <AdNetworkSettings adNetwork={adNetwork} handleUpdate={mockUpdate} />
+  it('should render ad management settings area with ad network dropdown button and helper text by default', function () {
+    const { getByText, getByRole } = renderWithProviders(
+      <AdManagement
+        adNetwork={adNetwork}
+        updateSettings={mockUpdate}
+        publisherId={''}
+        adSenseSlotId={''}
+        adManagerSlotId={''}
+      />
```

Comment:
nit: we don't need string interpolation here
```suggestion
        publisherId=""
        adSenseSlotId=""
        adManagerSlotId=""
      />
```